src/config/chainEndpoints.ts

```diff
@@ -4,6 +4,7 @@ interface ChainProvider {
   displayName: string;
   info?: string;
   endpoint: string;
+  favicon: string;
 }
 
 export const providerEndpoints: ChainProvider[] = [
```

Comment:
Since we will be adding more as we work, what about keeping the list of endpoints as a `json` file object and use the network name as the key. Similar to this scheme https://github.com/PlasmNetwork/api/blob/master/packages/types/src/networkSpecs/specs.ts#L2
In this case, the value of the `networkAlias` will be the name of the key.